INSTALL.rst

```diff
@@ -0,0 +1,363 @@
+.. The source of this document is INSTALL. During the doc build process,
+.. this file is copied over to doc/users/installing.rst.
+.. Therefore, you must edit INSTALL, *not* doc/users/installing.rst!
+.. _pip: https://pypi.python.org/pypi/pip/
+
+==========
+Installing
+==========
+
+.. note::
+
+    If you wish to contribute to the project, it's recommended you
+    :ref:`install the latest development version<install_from_source>`.
+
+
+
+.. contents.. 
+
+Installing an official release
+==============================
+
+Matplotlib and most of its dependencies all available as wheel packages for
+OSX, windows and many linux distributions::
```

Comment:
"and Linux distributions" (sure manylinux technically doesn't cover very old distros but I guess very old macos are likewise not supported too...)
(capitalize Linux for consistency with below)